<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load CSV</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboAlgo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>First fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Best fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Worst fit</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExport">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Export CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStress">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stress test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editBase">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit base</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSize">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddBlock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveBlocks">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadBlocks">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Block</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
